platform-core/src/main/java/com/webank/wecube/platform/core/service/PluginInstanceService.java

```diff
@@ -222,16 +222,19 @@ private DatabaseInfo handleCreateDatabase(Set<PluginPackageRuntimeResourcesMysql
             throw new WecubeCoreException("Only allow to plugin apply one s3 bucket so far");
         }
 
-        List<PluginMysqlInstance> mysqlInstances = pluginMysqlInstanceRepository
-                .findBySchemaNameAndStatus(pluginPackage.getName(), PluginMysqlInstance.MYSQL_INSTANCE_STATUS_ACTIVE);
-        if (mysqlInstances.size() > 0) {
-            PluginMysqlInstance mysqlInstance = mysqlInstances.get(0);
-            ResourceServer resourceServer = mysqlInstance.getResourceItem().getResourceServer();
-            return new DatabaseInfo(resourceServer.getHost(), resourceServer.getPort(), mysqlInstance.getSchemaName(),
-                    mysqlInstance.getUsername(), mysqlInstance.getPassword(), mysqlInstance.getResourceItemId());
-        } else {
-            return initMysqlDatabaseSchema(mysqlInfoSet, pluginPackage);
+        List<PluginPackage> sameNamePackages = pluginPackageRepository.findAllByName(pluginPackage.getName());
```

Comment:
Adding a method in PluginMysqlInstanceRepository can make the logic simpler.

    List<PluginMysqlInstance> findByStatusAndPluginPackage_name(String status, String packageName);